### ISSUE TITLE
using i18n.js with teaspoon ?

### ISSUE THREAD

**Issue by jvenezia** [fnando/i18n-js#294]
I'm using [teaspoon](https://github.com/modeset/teaspoon) with qunit to test my ember.js application, which uses i18n.js for internationalization.

When I run tests I get :
```
  Uncaught ReferenceError: require is not defined
```
It is called in this file, line 6.

When I manually change file and call **factory(this.I18n)** directly, it works.

Any clue on this? What would be the proper way to make it work ?
Thanks!

**Comment by jvenezia** [comment]
The problem seems to be [this condition](https://github.com/fnando/i18n-js/blob/master/app/assets/javascripts/i18n/filtered.js.erb#L4) .
It checks the **module** object from the global namespace. And actually **module** is used by Qunit, so it breaks the condition that should be false.

What is **module** ?

**Comment by fnando2** [comment]
That's for Node.js. I think the check must/can be improved. Maybe something like this:

if (typeof module !== 'undefined' && module.exports)

**Comment by fnando** [comment]
That's for Node.js. I think the check must/can be improved. Maybe something like this:

```javascript
if (typeof module !== 'undefined' && module.exports) {}
```

**Comment by jvenezia** [comment]
Yep, it does this [here](https://github.com/fnando/i18n-js/blob/master/app/assets/javascripts/i18n.js#L16) .

**Comment by fnando** [comment]
Fixed by 9453118d2914c7cf1272be34919be01b407af595

**Comment by jvenezia** [comment]
Thank's !
